lib/livebook/teams.ex

```diff
@@ -171,4 +172,84 @@ defmodule Livebook.Teams do
   defp add_org_errors(%Ecto.Changeset{} = changeset, errors_map) do
     Requests.add_errors(changeset, Org.__schema__(:fields), errors_map)
   end
+
+  @doc """
+  Returns an `%Ecto.Changeset{}` for tracking deployment group changes.
+  """
+  @spec change_deployment_group(DeploymentGroup.t(), map()) :: Ecto.Changeset.t()
+  def change_deployment_group(%DeploymentGroup{} = deployment_group, attrs \\ %{}) do
+    DeploymentGroup.changeset(deployment_group, attrs)
+  end
+
+  @doc """
+  Updates deployment group with the given changes.
+  """
+  @spec update_deployment_group(DeploymentGroup.t(), map()) ::
+          {:ok, DeploymentGroup.t()} | {:error, Ecto.Changeset.t()}
+  def update_deployment_group(%DeploymentGroup{} = deployment_group, attrs) do
+    deployment_group
+    |> DeploymentGroup.changeset(attrs)
+    |> Ecto.Changeset.apply_action(:update)
```

Comment:
Sorry, I think this function we can move to the view, as it is fine to call `apply_action` there, but I would keep `change_deployment_group`.

Reply:
FTR we do this for secrets and file systems too, so we should mirror for these too.

Reply:
Yeah, I am on it.